packages/effect/src/internal/doNotation.ts

```diff
@@ -16,6 +16,8 @@ type FlatMap<F extends TypeLambda> = {
   ): Kind<F, R1 & R2, O1 | O2, E1 | E2, B>
 }
 
+export type EmptyScope<F extends TypeLambda> = Kind<F, never, never, never, {}>
```

Comment:
```suggestion
/** @internal */
export type EmptyScope<F extends TypeLambda> = Kind<F, unknown, never, never, {}>
```

Reply:
in the end this type seems superfluous to me

Reply:
@gcanti, I've gotten rid of it.